configman/config_manager.py

```diff
@@ -825,6 +880,8 @@ def _setup_admin_options(self, values_source_list):
     #--------------------------------------------------------------------------
     def _walk_config_copy_values(self, source, destination, mapping_class):
         for key, val in source.items():
+            if key.endswith('$'):
```

Comment:
Why endswith? Why not startswith? Maybe I'm colored by having used jQuery a lot :) but I often see dollar signs as a prefix to indicate some sort key exception.